app/events/applications/application_deleted_event.rb

```diff
@@ -4,12 +4,16 @@ class Applications::ApplicationDeletedEvent < ApplicationRelatedEvent
 
   # @param [Cinstance] application
   def self.create(application)
+    service_id = application.service_id
     new(
       application: MissingModel::MissingApplication.new(id: application.id),
+      service_id: service_id,
+      user_key: application.user_key,
```

Comment:
Do we need the user_key?

Reply:
Unfortunately, it seems so. This is from the master branch:
https://github.com/3scale/porta/blob/be996712fbab0ca0f762a6e3676c9916204d17b6/app/lib/backend/model_extensions/cinstance.rb#L73-L76
https://github.com/3scale/porta/blob/be996712fbab0ca0f762a6e3676c9916204d17b6/app/lib/backend/model_extensions/cinstance.rb#L14